gateway/src/shard/event.rs

```diff
@@ -62,6 +62,8 @@ bitflags! {
         const GUILD_INTEGRATIONS_UPDATE = 1 << 13;
         /// A guild was updated.
         const GUILD_UPDATE = 1 << 14;
+        const INVITE_CREATE = 1 << 60; // TODO: Replace will actual number
+        const INVITE_DELETE = 1 << 61; // TODO: same as above
```

Comment:
They should just continue from the last number, this is only to make it a bitmap so we can filter for them in `some_events`

Reply:
Alright, I just set them to the next number after 45 (the highest number used before this)